.gitignore

```diff
@@ -56,5 +56,3 @@ public/css/main*
 webpack-bundle-stats.html
 server/rev-manifest.json
 google-credentials.json
-
-docker-compose.yml
```

Comment:
Get this file back under source control. If people want override files added to gitignore they can come up with a patter to ignore, but I'm still have not heard a good use case for it.